### ISSUE TITLE
space-after-type-colon false-positive on return of function types.

### ISSUE THREAD

**Issue by dikmax** [gajus/eslint-plugin-flowtype#32]
Given this code

```javascript
function aa(): (() => void) {
    
}
```

and config

```yaml
flowtype/space-after-type-colon: ["warn", "always"]
```

Shows error "There must be 1 space after return type colon". But there is space.

**Comment by gajus** [comment]
@dikmax Looks like a bug. Please have a look at the code base. It should be a simple fix.
